lib/tasks/reset_permissions.rake

```diff
@@ -22,6 +22,7 @@ namespace :permissions do
       user.password = password
       if user.save
         puts "Reset to user: #{user.login}, password: #{password}"
+        puts "WARNING: You need to change the password in /root/.hammer/cli.modules.d/foreman.yml manually!"
```

Comment:
Well, this is true for the `root` user. But since this task allows reseting of password for any user, this message should be more general.

Reply:
Yes, thank you for the comment.